packages/cli/src/commands/releasegold/transfer.ts

```diff
@@ -0,0 +1,36 @@
+import { displaySendTx } from '../../utils/cli'
+import { Flags } from '../../utils/command'
+import { ReleaseGoldCommand } from './release-gold'
+
+export default class Transfer extends ReleaseGoldCommand {
+  static description = 'Transfer stable tokens from the given contract address'
```

Comment:
`Dollars may be accrued to the ReleaseGold contract via validator epoch rewards.`